tap_github/__init__.py

```diff
@@ -1172,6 +1173,36 @@ def do_sync(config, state, catalog):
 
                 singer.write_state(state)
 
+def generate_jwt_token(github_app_id, github_private_key, expiration_time = 600, algorithm = 'RS256'):
+    actual_time = int(time.time())
+
+    payload = {
+        'iat': actual_time,
+        'exp': actual_time + expiration_time,
+        'iss': github_app_id
+    }
+
+    return jwt.encode(payload, github_private_key, algorithm=algorithm)
+
+def generate_access_token(config):
+    jwt_token = generate_jwt_token(
+        config['github_app_id'],
+        config['github_private_key']
+    )
+
+    headers = {
+        "Authorization": "Bearer {}".format(jwt_token),
+        "Accept": "application/vnd.github.machine-man-preview+json"
```

Comment:
Are you sure this is still needed? I think the API change graduated. See https://github.com/google/go-github/issues/1616